packages/patternfly-4/react-core/src/components/Progress/Progress.js

```diff
@@ -82,6 +82,7 @@ class Progress extends Component {
           getModifier(styles, variant, ''),
           getModifier(styles, measureLocation, ''),
           getModifier(styles, measureLocation === ProgressMeasureLocation.inside ? ProgressSize.lg : size, ''),
+          !title && getModifier(styles, 'singleline', ''),
```

Comment:
this should also work when title defaults to empty string

Reply:
Will update. I changed the default of title to null, though a user could still pass an empty string.

Reply:
What I meant was I think you can leave the default to empty string, because in js `!'' === !null`. So shouldn't have to add additional null checks

Reply:
Oh I see, good catch. I'll update it